packages/plugin-clientportal-api/src/graphql/resolvers/queries/clientPortal.ts

```
@@ -191,11 +192,13 @@ const configClientPortalQueries = {
       searchValue,
       topicId,
       isPrivate,
+      slug,
     }: {
       searchValue?: string;
       categoryIds: string[];
       topicId?: string;
       isPrivate: Boolean;
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)